docs/create_topology.md

```diff
@@ -186,7 +186,7 @@ To manually apply the controller run the following command after cloning the
 Keysight controller yaml:
```

Comment:
and this line

Reply:
sure, also removed the notice since ixiatg operator images are publicly available